index.js

```diff
@@ -144,7 +144,7 @@ Osm.prototype.put = function (id, element, opts, cb) {
   var errs = checkElement(element, 'put')
   if (errs.length) return cb(errs[0])
 
-  var doc = Object.assign({}, element, { id: id, timestamp: new Date().toISOString() })
+  var doc = Object.assign({ timestamp: new Date().toISOString() }, element, { id: id })
```

Comment:
actually looking at this now, I notice that `created_at` is not set by kappa-osm, but maybe it should? (it is currently set by mapeo-server I think). I think `created_at` should maybe be over-rideable by the user, whereas `timestamp` shouldn't. `timestamp` == datetime the specific version was written on the server (effectively, modified time), `created_at` == datetime record was first created e.g. shouldn't change (but might need to be changed if the user's device time was incorrect), and might be set in the front end, e.g. when the user first creates an observation rather than when they save it.

Apologies for thinking out loud here... I think this line should maybe be:

```js
var doc = Object.assign(
  { created_at: new Date().toISOString() },
  element,
  { id: id,  timestamp: new Date().toISOString() }
)
```

E.g. server controls timestamp, user can override `created_at`. What do you think @noffle @okdistribute ?